fem/linearform.cpp

```diff
@@ -69,12 +78,38 @@ void LinearForm::Assemble()
             AddElementVector (vdofs, elemvect);
          }
       }
-
+   }
    AssembleDelta();
 
    if (blfi.Size())
+   {
+      Mesh *mesh = fes->GetMesh();
+
+      // Which boundary attributes need to be processed?
+      Array<int> bdr_attr_marker(mesh->bdr_attributes.Size() ?
+                                 mesh->bdr_attributes.Max() : 0);
+      bdr_attr_marker = 0;
+      for (int k = 0; k < blfi.Size(); k++)
+      {
+         if (blfi_marker[k] == NULL)
+         {
+            bdr_attr_marker = 1;
+            break;
+         }
+         Array<int> &bdr_marker = *blfi_marker[k];
+         MFEM_ASSERT(bdr_marker.Size() == bdr_attr_marker.Size(),
+                     "invalid boundary marker for boundary face integrator #"
```

Comment:
Minor edit: I would change this comment from "... boundary face integrator" to "... boundary integrator"